pandas/_libs/lib.pyx

```diff
@@ -2232,7 +2232,8 @@ def maybe_convert_objects(ndarray[object] objects, bint try_float=0,
     return objects
 
 
-_no_default = object()
+# Note: _no_default is exported to the public API in pandas.api.extensions
+_no_default = object()  #: Sentinel indicating the default value.
```

Comment:
Should we rename this to `no_default` ? (no leading underscore)

Reply:
I thought about it, don't have a strong opinion.

Reply:
i agree we should just rename this

Reply:
note that we have a number of type we use object as the marker (so should change those in a followup)